web/src/containers/StatePersonnel.js

```diff
@@ -0,0 +1,111 @@
+
+import Dollars from '../components/Dollars';
+import PersonnelForm from '../components/FormPersonnel';
+
+class StatePersonnel extends Component {
+  showResults = data => {
+    console.log(data);
+  };
+
+  render() {
+    const { goTo } = this.props;
+    const { statePersonnel, contractingPersonnel } = this.props;
+
+    const stateNextTotal = statePersonnel.reduce(
+      (total, personnel) =>
+        total + ((personnel.nextCompensation * personnel.nextTime) / 100),
+      0
+    );
+    const stateNextNextTotal = statePersonnel.reduce(
+      (total, personnel) =>
+        total +
+        ((personnel.nextCompensation * personnel.nextNextTime * 1.03) / 100),
+      0
+    );
+    const contractingNextTotal = contractingPersonnel.reduce(
+      (total, personnel) =>
+        total + ((personnel.nextCompensation * personnel.nextTime) / 100),
+      0
+    );
+    const contractingNextNextTotal = contractingPersonnel.reduce(
+      (total, personnel) =>
+        total +
+        ((personnel.nextCompensation * personnel.nextNextTime * 1.03) / 100),
+      0
+    );
+
+    return (
+      <Box py={4}>
+        <Heading>Personnel costs for Administration</Heading>
+        Most activities include costs for state and contracting personnel. You
+        may also want to hire consultants for legal services or outreach, for
+        example. We’ll go over other miscellaneous expenses like software
+        licenses and supplies in the next section.
+        <table>
+          <tbody>
+            <tr>
+              <td>
+                <li />
+              </td>
+              <td>State personnel</td>
+              <td>2019</td>
+              <td><Dollars value={stateNextTotal} hideCents /></td>
+              <td>2020</td>
+              <td><Dollars value={stateNextNextTotal} hideCents /></td>
+            </tr>
+            <tr>
+              <td>
+                <li />
+              </td>
+              <td>Contracting personnel</td>
+              <td>2019</td>
+              <td><Dollars value={contractingNextTotal} hideCents /></td>
+              <td>2020</td>
+              <td><Dollars value={contractingNextNextTotal} hideCents /></td>
+            </tr>
+          </tbody>
+        </table>
+        {/* <Form onSubmit={this.showResults} /> */}
+        <Heading>State personnel</Heading>
+        <PersonnelForm personnelType="state" />
+        <Divider my={4} color="gray2" />
+        <Heading>Contracting personnel</Heading>
+        <PersonnelForm personnelType="contracting" />
+        <Divider my={4} color="gray2" />
+        <Button onClick={() => goTo('/state-start')}>Continue</Button>
+      </Box>
+    );
+  }
+}
+
+StatePersonnel.propTypes = {
+  goTo: PropTypes.func.isRequired,
+  statePersonnel: PropTypes.array.isRequired,
+  contractingPersonnel: PropTypes.array.isRequired
+};
+
+const mapDispatchToProps = dispatch =>
+  bindActionCreators({ goTo: path => push(path) }, dispatch);
```

Comment:
we should figure out if there's a better way to abstract this `goTo` method across our connected components so we don't have to repeat it everywhere (I know I started this 😳) but we can still navigate to diff pages with redux actions

Reply:
Agreed.  It'd be pretty cool if we mapped out the flow in a global object somewhere and then had a `next` event that advanced to the next page, maybe?  Or something else, I dunno.  lol.  We can brainstorm on a good approach!